src/features/slop/ui/slop-status-list.tsx

```diff
@@ -45,8 +50,8 @@ const SlopStatusList = ({ list }: SlopStatusListProps) => {
         {list.map((item, index) => (
           <tr
             key={index}
```

Comment:
여기에도 key를 쓰면 될 것 같아~
```suggestion
            key={item.key}
```